src/purchases.ts

```diff
@@ -1118,6 +1140,12 @@ export default class Purchases {
     }
   }
 
+  private static async throwIfIOSPlatform() {
+    if (Platform.OS === "ios") {
```

Comment:
Probably yes but just making sure, this will include iPad OS right?

Reply:
I believe so, the only options are `android` and `ios` https://reactnative.dev/docs/platform#os